concepticondata/conceptlists/BirketSmith-1928-510.tsv

```diff
@@ -0,0 +1,511 @@
+BirketSmith-1928-510-159	159	Chin	1510	CHIN	Man and the Human Body
+BirketSmith-1928-510-160	160	Cunnus	1837	VULVA	Man and the Human Body
+BirketSmith-1928-510-161	161	Ear	1247	EAR	Man and the Human Body
+BirketSmith-1928-510-162	162	Elbow	981	ELBOW	Man and the Human Body
+BirketSmith-1928-510-163	163	Eye	1248	EYE	Man and the Human Body
+BirketSmith-1928-510-164	164	Face	1560	FACE	Man and the Human Body
+BirketSmith-1928-510-165	165	Finger, thumb	1781	THUMB	Man and the Human Body
+BirketSmith-1928-510-166	166	Finger, fore	577	FOREFINGER	Man and the Human Body
+BirketSmith-1928-510-167	167	Finger, middle	3387	MIDDLE FINGER	Man and the Human Body
+BirketSmith-1928-510-168	168	Finger, ring			Man and the Human Body
+BirketSmith-1928-510-169	169	Finger, little	2774	LITTLE FINGER	Man and the Human Body
+BirketSmith-1928-510-170	170	Foot	1301	FOOT	Man and the Human Body
+BirketSmith-1928-510-171	171	Hair (plur.)	1040	HAIR	Man and the Human Body
+BirketSmith-1928-510-172	172	Hand	1277	HAND	Man and the Human Body
+BirketSmith-1928-510-173	173	Head	1256	HEAD	Man and the Human Body
+BirketSmith-1928-510-174	174	Heart	1223	HEART	Man and the Human Body
+BirketSmith-1928-510-175	175	Heel	980	HEEL	Man and the Human Body
+BirketSmith-1928-510-176	176	Knee	1371	KNEE	Man and the Human Body
+BirketSmith-1928-510-177	177	Leg	1297	LEG	Man and the Human Body
+BirketSmith-1928-510-178	178	Man (homo)	683	PERSON	Man and the Human Body
+BirketSmith-1928-510-179	179	Man (vir)	1554	MAN	Man and the Human Body
+BirketSmith-1928-510-180	180	Mouth	674	MOUTH	Man and the Human Body
+BirketSmith-1928-510-181	181	Nail	1896	FINGERNAIL OR TOENAIL	Man and the Human Body
+BirketSmith-1928-510-182	182	Navel	1838	NAVEL	Man and the Human Body
+BirketSmith-1928-510-183	183	Neck	1333	NECK	Man and the Human Body
+BirketSmith-1928-510-184	184	Neck, back of	1347	NAPE (OF NECK)	Man and the Human Body
+BirketSmith-1928-510-185	185	Nose	1221	NOSE	Man and the Human Body
+BirketSmith-1928-510-186	186	Penis	1222	PENIS	Man and the Human Body
+BirketSmith-1928-510-187	187	Posterior	834	BUTTOCKS	Man and the Human Body
+BirketSmith-1928-510-188	188	Testicle	797	TESTICLES	Man and the Human Body
+BirketSmith-1928-510-189	189	Thigh	471	UPPER LEG (THIGH)	Man and the Human Body
+BirketSmith-1928-510-190	190	Toe	1389	TOE	Man and the Human Body
+BirketSmith-1928-510-191	191	Tooth	1380	TOOTH	Man and the Human Body
+BirketSmith-1928-510-192	192	Woman	962	WOMAN	Man and the Human Body
+BirketSmith-1928-510-193	193	Airhole in the house			Dwellings and Furniture
+BirketSmith-1928-510-194	194	Camp, settlement			Dwellings and Furniture
+BirketSmith-1928-510-195	195	Door-opening			Dwellings and Furniture
+BirketSmith-1928-510-196	196	Drying frame			Dwellings and Furniture
+BirketSmith-1928-510-197	197	Fire drill	517	FIREDRILL	Dwellings and Furniture
+BirketSmith-1928-510-198	198	Fire stone			Dwellings and Furniture
+BirketSmith-1928-510-199	199	House, stone			Dwellings and Furniture
```

Comment:
I think I'd prefer to leave it unmapped since the fact that it is made of stone (e.g. instead of wood, not only as opposition to the snow house) is central part of the gloss.

Reply:
I agree.